### ISSUE TITLE
how to get rid of this debug message: No handler matched but default handler present, added it.

### ISSUE THREAD

**Issue by cyberic99** [attwad/python-osc#83]
I can't find a way to disable this message

Even if I put dispatcher.map("/*", recv_osc_event), it is still printed...

Thank you

**Comment by attwad** [comment]
This comes from 
https://github.com/attwad/python-osc/blob/0217beab225096d67e0fa442eaf3f095ea64c0bb/pythonosc/dispatcher.py#L80

Do you need the default handler? if you don't need it then this message should go away
```set_default_handler(None)``` clears it as per https://github.com/attwad/python-osc/blob/0217beab225096d67e0fa442eaf3f095ea64c0bb/pythonosc/dispatcher.py#L116

**Comment by cyberic99** [comment]
> Do you need the default handler? 

Yes, I'd like to keep it, because I do the matching later myself.

I thought I could try to match on /* instead, but it does not work

**Comment by attwad** [comment]
Interesting, what kind of address are you getting that doesn't match `/*` ? because a unit test exists for this and `/*` works fine with ```"/anything/matches"``` for example

https://github.com/attwad/python-osc/blob/0217beab225096d67e0fa442eaf3f095ea64c0bb/pythonosc/test/test_dispatcher.py#L101
